packages/common/src/api/comments.ts

```diff
@@ -7,9 +7,26 @@ import { ID } from '~/models'
 import { decodeHashId, encodeHashId } from '~/utils'
 
 // Helper method to save on some copy-pasta
+// Updates the array of all comments
+const optimisticUpdateCommentList = (
+  entityId: number,
+  updateRecipe: (prevState: Comment[] | undefined) => Comment | void,
```

Comment:
Should the updateRecipe return an array of Comments?

Reply:
True! Although tbh I don't fully understand the proxy object stuff so idk if returning a new array properly works lol. Will update the type tho